src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-policy-details/rgw-multisite-sync-policy-details.component.html

```diff
@@ -118,6 +110,6 @@
 <ng-template #deleteTpl>
   <cd-alert-panel type="danger"
                   i18n>
-    Are you sure you want to delete these Flow?
+    Are you sure you want to delete these {{ resourceType | upperFirst }}?
   </cd-alert-panel>
```

Comment:
I always dislike adding redundancy on the texts displayed, no one likes to read twice the same thing for no reason. The default text for the delete modal already asks ` Are you sure you want to delete` so if we really think the `cd-alert-panel` is necessary I think it should have a descriptive text of why this operation is dangerous